ext/odbc/tests/odbc_field_len_001.phpt

```diff
@@ -0,0 +1,43 @@
+--TEST--
+odbc_field_len(): Getting the length of the field
+--EXTENSIONS--
+odbc
+--SKIPIF--
+<?php include 'skipif.inc'; ?>
+--FILE--
+<?php
+
+include 'config.inc';
+
+$conn = odbc_connect($dsn, $user, $pass);
+
+odbc_exec($conn, 'CREATE TABLE field_len (foo INT, bar TEXT, baz VARBINARY(50))');
+odbc_exec($conn, "INSERT INTO field_len VALUES (1, 'bar', CONVERT(VARBINARY(50), 'baz'))");
+
+$res = odbc_exec($conn, 'SELECT * FROM field_len');
+try {
+    odbc_field_len($res, 0);
+} catch (ValueError $error) {
+    echo $error->getMessage() . "\n";
+}
+var_dump(odbc_field_len($res, 1));
+var_dump(odbc_field_len($res, 2));
+var_dump(odbc_field_len($res, 3));
+var_dump(odbc_field_len($res, 4));
+
+odbc_close($conn);
+?>
+--CLEAN--
+<?php
+require 'config.inc';
+$conn = odbc_connect($dsn, $user, $pass);
+odbc_exec($conn, 'DROP TABLE field_len');
+?>
+--EXPECTF--
+odbc_field_len(): Argument #2 ($field) must be greater than 0
+int(10)
+int(2147483647)
+int(50)
+
+Warning: odbc_field_len(): Field index larger than number of fields in %s on line %d
```

Comment:
It even warns :o truly an extension that received no love for a long time to be this inconsistent

Reply:
Yeah, I guess this is because providing a column with a non-positive value is surely a programmatic error, since there is no 0 column. On the other hand, you cannot be sure wha the maximum number of fields really is, that's why this is just a warning.

Reply:
One should know the database schema one is dealing with tho?

Also ``odbc_num_fields()`` exists, but that's an aside.